cmd/eyes/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"os"
+
+	"github.com/tendermint/tmlibs/cli"
+
+	"github.com/tendermint/basecoin"
+	"github.com/tendermint/basecoin/cmd/basecoin/commands"
+	"github.com/tendermint/basecoin/modules/base"
+	"github.com/tendermint/basecoin/modules/etc"
+	"github.com/tendermint/basecoin/stack"
+)
+
+// BuildApp constructs the stack we want to use for this app
+func BuildApp() basecoin.Handler {
+	return stack.New(
+		base.Logger{},
+		stack.Recovery{},
+	).
```

Comment:
Just to clear my understanding - _eyes_ will run concurrently to a basecoin instance and therefor needs it's own stack and logger